.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
I disagree. I would have the `node_modules` and `bower_components` in the project's `.gitignore`. 

I see global ignore as user or platform specific - Mac has different gitignores (`.DS_Store`, etc), your person editor (Xcode, Vim, etc) and those go into global gitignore. 

The local gitignore for repositories are for the project itself, and platform agnostic -- because we are using Node.js therefore we ignore `node_modules`. I don't think this should be in your global gitignore. While it may end up being that all of your projects have node_modules in the gitignore I see this as basic scaffolding in the same way we create a package.json file for Node.js projects. A new contributor should be able to simply download the repository and get going without having to change their global gitignore -- the project itself should know internally what needs to be ignored. In the same way we may have different coding standards in `.editorconfig` or `.jshintrc` files, these are project specific.
Regardless that these will almost always be the same, I see this as a very project specific thing and should be isolated in that way.

Just my quick thoughts.